simpledisplay.d

```diff
@@ -770,6 +808,8 @@ interface->SetProgressValue(hwnd, 40, 100);
 		simpledisplay was stand alone until about 2015. It then added a dependency on [arsd.color] and changed its name to `arsd.simpledisplay`.
 
 		On March 4, 2023 (dub v11.0), it started importing [arsd.core] as well, making that a build-time requirement.
+
+		On October 5, 2024, apitrace support was added for Linux targets.
```

Comment:
“for non-OSX Posix targets” sounded needlessly complicated.

Reply:
And given that I haven’t tested this on FreeBSD or similar, it would be a stretch to say it’s “supported” there, anyway.